storage/migrations/02_oasis_3_emerald.up.sql

```diff
@@ -35,6 +35,8 @@ CREATE TABLE oasis_3.emerald_transactions
   -- later store sufficiently detailed data in other columns or if we turn out
   -- to be able to get a copy of the transaction elsewhere.
   raw         BYTEA NOT NULL,
+  -- result_raw is cbor(CallResult).
+  result_raw  BYTEA NOT NULL,
```

Comment:
(Just a thought, not a suggestion to change anything in this PR)

Might be nice to have both of these be the native `JSON` type in postgres. We'd gain human-readable column values, and the ability to perform ad-hoc queries/analyses over "raw" data in psql. OTOH we risk losing some esoteric cbor nuance that gets lost when converting to json.

Reply:
that might be nice. also if someone could come up with a way to deep parse RawMessage innards, e.g. the tx in the utx, and the body in the tx and get it all into json. similarly for the result

Reply:
Agreed. Not in this PR though, right? Filed #207 

> the tx in the utx
That what your [`OpenUtxNoVerify()`](https://github.com/oasisprotocol/oasis-indexer/blob/09dd2b5dd4a74d381ead68151ac0d42ec6aa5c59/analyzer/uncategorized/runtimes.go#L17-L17), I thought? Are there other special cases besides evm that we need to cover?

> the body in the tx
that's just cbor, so it's easy to decode generically, right?

Reply:
oh shoot, ethereum format transactions :thinking:

Reply:
Yeah for eth format we'd have to either store the JSON version of [`decodeEthRawTx()`](https://github.com/oasisprotocol/oasis-indexer/blob/09dd2b5dd4a74d381ead68151ac0d42ec6aa5c59/analyzer/uncategorized/evm_raw_tx.go#L14), or else store both the raw byte[] _and_ the JSON ...